web/src/pages/report/reportIndex.ts

```diff
@@ -0,0 +1,45 @@
+import OurDnaDataQuality from './reports/ourdna/DataQuality'
+import OurDnaDemographics from './reports/ourdna/Demographics'
+import OurDnaRecruitment from './reports/ourdna/Recruitment'
+import OurDnaProcessingTimes from './reports/ourdna/SampleProcessing'
+
+type ReportTab = {
+    title: string
+    id: string
+    content: React.ComponentType
+}
+
+type ReportDefinition = {
+    title: string
+    tabs: ReportTab[]
+}
+
+export const reports: Record<string, Record<string, ReportDefinition>> = {
+    ourdna: {
```

Comment:
Can we ask for clarification, just put a comment here, this is the place where we link project to custom reports. We've got lost a bit on this one.

Reply:
Yeah that's right, this is sort of an interim thing while we have custom reports but don't have them stored in the database, we need somewhere to store a listing of reports. The structure here works with the code here https://github.com/populationgenomics/metamist/blob/SET-238-dashboard-v-2-prototype/web/src/pages/report/ProjectReport.tsx#L9 which matches the params in the url to load the correct report.
It is also imported in the project summary page to show a list of reports for each project, matched on the project name.